localization/LANG_DE.h

```diff
@@ -248,13 +248,13 @@
 #define STR137_TEXT "Bedieneinheit P Status"
 #define STR138_TEXT "Servicegerät Status"
 #define STR140_TEXT "Alle Geräte löschen"
-#define STR500_TEXT "Mo"
-#define STR501_TEXT "Di"
-#define STR502_TEXT "Mi"
-#define STR503_TEXT "Do"
-#define STR504_TEXT "Fr"
-#define STR505_TEXT "Sa"
-#define STR506_TEXT "So"
+#define STR500_TEXT "Montag"
```

Comment:
Ulf, full-named days of the week is placed below: 
#define ENUM_WEEKDAY_01_TEXT "Montag"
#define ENUM_WEEKDAY_02_TEXT "Dienstag"
#define ENUM_WEEKDAY_03_TEXT "Mittwoch"
#define ENUM_WEEKDAY_04_TEXT "Donnerstag"
#define ENUM_WEEKDAY_05_TEXT "Freitag"
#define ENUM_WEEKDAY_06_TEXT "Samstag"
#define ENUM_WEEKDAY_07_TEXT "Sonntag"

Maybe it makes sense to leave abbreviated names for other purposes?

Reply:
Aaargh, sorry, didn't see that :(
Yes, abbreviations could make sense, of course. Can you or @fredlcore change that within the merge-process? Or should I edit the file within my forked repo and make a new PR? Sorry, it's my first PR (ever! ;) ), so I'm not yet familliar with that processes..

Reply:
..just changed it back to the abbreviations, hope the PR still works.. ;)

Reply:
..just spotted even more names which aren't fully (or in the correct way) written out (e.g. 8444), but I'll do that another time and make another PR then.